Documentation/WritingDocForExtension/Migrate.rst

```diff
@@ -32,32 +32,226 @@ a migration tool. This tool can be used to automatically create a
 :file:`Documentation/guides.xml` from the information contained in your
 :file:`Documentation/Settings.cfg`.
 
-`Docker <https://docs.docker.com/install/>`__ needs to be installed on your
+`Docker <https://docs.docker.com/install/>`__ (or a drop-in replacement like
+`Podman <https://podman.io>`__) needs to be installed on your
 operating system for the tool to work:
 
-..  code-block:: shell
+..  tabs::
+
+    ..  group-tab:: Docker
+
+        ..  code-block:: bash
+
+            docker run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+    .. group-tab:: Podman
+
+        ..  code-block:: bash
+
+            podman run --rm --pull always \
+              -v $(pwd):/project \
+              -it ghcr.io/typo3-documentation/render-guides:latest \
+              migrate Documentation
+
+
+The last parameter (:bash:`Documentation`) is the name of the directory, where your
+:file:`Settings.cfg` is currently placed in.
 
-    docker run --rm --pull always \
-        -v $(pwd):/project \
-        -it ghcr.io/typo3-documentation/render-guides:latest \
-        migrate Documentation
+After the migration is performed, you will see output about which settings were
+converted, if some old settings were discarded, or errors occurred.
 
 Try out the rendering locally
 =============================
 
 Use our Docker container to render your documentation locally. Read more about
 :ref:`local rendering <rendering-docs>`.
 
-Improve your documentation to render without warning
-====================================================
+..  _migrate-detailed-steps:
+
+Further steps to adapt to the new rendering
+===========================================
+
+You should perform the following tasks to conclude the migration to the
+new rendering tool:
+
+..  rst-class:: bignums
+
+1.  Improve your documentation to render without warning
+
+    Rendering your documentation should not yield any warnings or errors.
+
+    If you get error messages, often they refer to wrong indentation, missing
+    :ref:`interlinks <settings-guides-interlink-mapping>`, orphaned files or
+    outdated ReST identifiers.
+
+    If you are unable to address a warning/error with changes in your documentation
+    feel free to ask in Slack channel #typo3-documentation (see :ref:`how-to-get-help`).
+
+    If you believe you found a specific bug in the new PHP-based rendering, please open
```

Comment:
I would avoid "new" as it tends to stay there for a longer time. Just "in the PHP-based rendering" is sufficient.

Reply:
I agree; I only left "new" rendering in the top note, and removed it for other occurences.